### PR TITLE
Updated documentation for Windows setup

### PR DESCRIPTION
**Pull request by SurajBDeore**
Platform-Specific Setup Instructions
 
This update adds separate setup instructions for both Linux and Windows environments:
 
-  Windows: Added steps for installing required tools using the Scoop package manager, including `findutils` and `make`.

-  Linux: Clarified package installation for `build-essential` and `graphviz` using native package managers.
 
These changes simplify the setup process and ensure compatibility across platforms for contributors.

### CODE REVIEW

File: documentation/dev_setup.md [hunk 1]
```diff
@@ -1,60 +1,86 @@
...
+    ```
+  * you need to install GNU Make. You can do this by running the following command:
+ 
+    ```bash
+    scoop install make
+    ```
+  * You need to install Graphviz.
+ 
+    ```bash
+      scoop install Graphviz
```
Comment (christophkloeffel) [inline documentation/dev_setup.md:68]: please fix indentation

File: documentation/dev_setup.md [hunk 2]
```diff
@@ -1,60 +1,86 @@
...
+ 
   * Check out https://github.com/bmw-software-engineering/lobster and
     put the root of the repo on your `PYTHONPATH`.
-
+ 
   * Install from PyPI, carfully avoiding to install the TRLC
     dependency as that will make things really confusing:
-
+ 
     ```bash
```
Comment (christophkloeffel) [inline documentation/dev_setup.md:37]: please also fix indentation

File: documentation/dev_setup.md [hunk 3]
```diff
@@ -1,60 +1,86 @@
...
-
+ 
+# Common Setup for Windows and Linux
+ 
 * You need a suitable version of Python3 (3.8, 3.9, 3.10, or
   3.11). The reason we do not support 3.12 or later (for the dev
   setup) is the dependency on CVC5. Users could use 3.12; but they
   won't be able to use the CVC5 API.
-
+
```
Comment (christophkloeffel) [inline documentation/dev_setup.md:15]: please avoid cluttering the version control by adding spaces in blank lines

File: documentation/dev_setup.md [hunk 4]
```diff
@@ -1,60 +1,86 @@
...
-
-To contribute to TRLC you will need to be able to run the
-testsuite. Currently this is only realistic on GNU/Linux, although it
-may be possible to get it to work on other platforms.
-
+ 
+To contribute to TRLC, you will need to be able to run the
+testsuite. Currently, this is only realistic on GNU/Linux and Windows,
+although it may be possible to get it to work on other platforms.
+
```
Comment (christophkloeffel) [inline documentation/dev_setup.md:6]:
```suggestion
To contribute to TRLC, you will need to be able to run the
testsuite. Therefore you need GNU make and GNU findutils.
We do provide setup steps for Windows and Linux. If you run into any problems on macOS, make sure you install the latest GNU make version.
```

Comment (christophkloeffel) [reply documentation/dev_setup.md:6]: It is possible on macOS for sure. I think the only hindrance could be an outdated make version

**Review by christophkloeffel — CHANGES_REQUESTED** [review]

File: documentation/dev_setup.md [hunk 5]
```diff
@@ -42,6 +38,43 @@ may be possible to get it to work on other platforms.
...
+* You need GNU Make. This should be available on all sane GNU/Linux
+  distributions. On Debian the package is called `build-essential`.
+
+* You need to install Graphviz. On Debian the package is called
+  `graphviz`.
+
+# Windows Setup
+
+* You need to Install Scoop.
+  * Check out https://scoop.sh/
```
Comment (phiwuu) [inline documentation/dev_setup.md:52]: please remove the extra bullet point:
```
* You need to Install Scoop.
  Check out https://scoop.sh/
```

**Review by phiwuu — CHANGES_REQUESTED** [review]

**Review by christophkloeffel — APPROVED** [review]

**Review by phiwuu — APPROVED** [review]

**Pull request merged**